src/jaxsim/math/adjoint.py

```diff
@@ -137,11 +137,12 @@ def inverse(adjoint: jtp.Matrix) -> jtp.Matrix:
             jtp.Matrix: The inverse adjoint matrix.
         """
         A_X_B = adjoint
-        A_H_B = Adjoint.to_transform(adjoint=A_X_B)
 
-        A_R_B = A_H_B[0:3, 0:3]
-        A_o_B = A_H_B[0:3, 3]
+        A_R_B = A_X_B[0:3, 0:3]
 
-        return Adjoint.from_rotation_and_translation(
-            rotation=A_R_B, translation=A_o_B, inverse=True
+        return jnp.vstack(
+            [
+                jnp.block([A_R_B.T, -A_R_B.T @ A_X_B[0:3, 3:6] @ A_R_B.T]),
+                jnp.block([jnp.zeros(shape=(3, 3)), A_R_B.T]),
+            ]
         )
```

Comment:
I just realized that all these methods are correct only if they operate on transforms for 6D velocites, not those for 6D forces. Let's keep this in mind. We might also want to add an additional argument to turn on logic for 6D forces. I think that the term _adjoint_ only corresponds to the tangent of the original space ($\text{SE}(3) in this case), and it might mean that the considered adjoint is the trasform for 6D velocities. Not sure if the real terminology for the trasform for 6D forces is _dual_ adjoint.  cc @traversaro

Reply:
Murray Lee Sastry seems just to write of "adjoint transpose". Indeed Park calls it "dual adjoint" in https://mae.eng.uci.edu/~bobrow/papers_files/DynamicsParkBobrow.pdf , but if you want to avoid confusion in the reader you can also use more user friendly terms like "velocity transform" and "force transform" or something similar.